tox.ini

```diff
@@ -3,10 +3,9 @@ envlist = es, addons-devhub-editors, main, flake8, docs, assets
 
 [testenv]
 basepython = python2.6
-install_command = pip install --no-deps --exists-action=w --download-cache=/tmp/pip-cache --find-links https://pyrepo.addons.mozilla.org/ {opts} {packages}
+install_command = pip install --no-deps --exists-action=w --download-cache=/tmp/pip-cache --find-links https://pyrepo.addons.mozilla.org/wheelhouse/ {opts} {packages}
```

Comment:
If packages aren't in the wheelhouse, does it fall back to our mirror or the public one (or none?)?